llm_on_ray/finetune/finetune.py

```diff
@@ -130,6 +121,52 @@ def get_accelerate_environment_variable(config: Dict[str, Any]) -> dict:
     return mode_env_vars[device][accelerate_mode]
 
 
+def convert_to_training_args(cls, config):
+    device = config["Training"]["device"]
+    accelerate_mode = config["Training"]["accelerate_mode"]
+    checkpoint_dir = config["General"]["checkpoint_dir"]
+    save_strategy = "no" if checkpoint_dir is None else "epoch"
+
+    args = {
+        "output_dir": config["General"]["output_dir"],
+        "gradient_checkpointing": config["General"]["enable_gradient_checkpointing"],
+        "save_strategy": save_strategy,
+        "bf16": config["Training"]["mixed_precision"] == "bf16",
+        "num_train_epochs": config["Training"]["epochs"],
+        "per_device_train_batch_size": config["Training"]["batch_size"],
+        "per_device_eval_batch_size": config["Training"]["batch_size"],
+        "learning_rate": config["Training"]["learning_rate"],
+        "logging_steps": config["Training"]["logging_steps"],
+        "lr_scheduler_type": config["Training"]["lr_scheduler"],
+        "weight_decay": config["Training"]["weight_decay"],
+        "gradient_accumulation_steps": config["Training"]["gradient_accumulation_steps"],
```

Comment:
Do we have default values for all these configurations in `config`? Previously we wrote `config["Training"].get("gradient_accumulation_steps", 1)`

Reply:
yes, this option has default value 1 in finetune_config.py